openstackclient/Containerfile

```diff
@@ -51,6 +51,8 @@ WORKDIR /configuration
 
 VOLUME ["/configuration"]
 
+ENV CLIFF_FIT_WIDTH=1
```

Comment:
This should be the default already, I assume it is more of an issue to detect the actual terminal width when running inside a container?